src/coreclr/vm/typedesc.cpp

```diff
@@ -1730,6 +1730,11 @@ BOOL TypeVarTypeDesc::SatisfiesConstraints(SigTypeContext *pTypeContextOfConstra
     }
     CONTRACTL_END;
 
+    // During EEStartup, we cannot safely validate constraints, but we can also be confident that the code doesn't violate them
+    // Just skip validation and declare that the constraints are satisfied.
+    if (g_fEEInit)
```

Comment:
@davidwrighton, this is still required, correct?

Reply:
Yes. That is required to handle constraints existing on interfaces that are applied to primitive types.